connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -430,7 +431,9 @@ public Void call() throws Exception {
                         if (!configState.contains(connName)) {
                             callback.onCompletion(new NotFoundException("Connector " + connName + " not found"), null);
                         } else {
-                            callback.onCompletion(null, new ConnectorInfo(connName, configState.connectorConfig(connName), configState.tasks(connName)));
+                            callback.onCompletion(null, new ConnectorInfo(connName, configState.connectorConfig(connName),
```

Comment:
nit: Our latest code style (which is not currently strictly enforced in AK Connect code) suggests to put every argument in its own line in the presence of multiline calls. You might want to apply this pattern in new changes, even if the checkstyle is not enforced right now.

Reply:
I think it applies to the declaration of methods, not the calls to the method.

Reply:
I'd say calls as well. It's very common to apply similar style to both. In any case, with this still being a nitpick, I think it makes sense to have 1) all args in one line (not flexible) 2) args split in exactly as many lines are needed by the width - which leaves several args per line and 3) have 1 arg per line. There's not too much uniformity in AK right now, but wanted to let you know of a trend.

Reply:
Ok, it looks better now. Let's leave it this way, with two lines.